src/schemas/db/node-contacts.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+module.exports = connection => {
+  const NodeContactSchema = new Schema(
+    {
+      // Handling of different types of contact
+      type: {
+        type: Schema.Types.String,
+        enum: ['direct-bluethooth', 'direct-network', 'indirect', 'manual'],
```

Comment:
I think this is a typo `direct-bluethooth`.

Reply:
Didn't noticed. Will fix it right away.